near-campaign/src/get_keys.rs

```diff
@@ -1,28 +1,25 @@
 use crate::*;
 
 /*
-  get_keys allows us to check the status of specific keys -
-  if a key was claimed or it is still active etc
- */
+ get_keys allows us to check the status of specific keys -
+ if a key was claimed or it is still active etc
+*/
 
 #[derive(Serialize, Deserialize)]
 #[serde(crate = "near_sdk::serde")]
 pub struct Key {
   pk: Base58PublicKey,
-  status: KeyStatus,
+  status: Option<KeyStatus>,
 }
 
 #[near_bindgen]
 impl Campaign {
   pub fn get_keys(&self, keys: Vec<Base58PublicKey>) -> Vec<Key> {
     keys
-      .iter()
+      .into_iter()
```

Comment:
Why `into_iter` is more efficient than `iter`? Because it transfers the ownership instead of creating a reference?

Reply:
Yeah avoids cloning the data from within the function. No reason not to consume the vector since it's not used after. Functionally the same